components/31-molecules/file-upload/file-upload.functions.js

```diff
@@ -44,12 +44,21 @@
       elem.dataset.file = input.files[0].name;
     };
 
+    /**
+     * Make container element focusable instead of the file input.
+     */
+    const initFocus = () => {
```

Comment:
It's best not to change this.
It'll break keyboard functionality and screenreaders can miss vital information that's embedded in the input element.

I suggest using :focus-within in combination with some js magic for Edge support.
Or changing the template so we can use the sibling selector.

Reply:
Template has been changed 🙃